nautobot_data_validation_engine/jobs.py

```diff
@@ -47,6 +61,10 @@ class RunRegisteredDataComplianceRules(Job):
         description="Not selecting any rules will run all rules listed.",
     )
 
+    run_existing_rules_in_report = BooleanVar(
+        label="Run existing validation rules", description="Include manually created data validation rules in report?"
```

Comment:
Let's refer to the regex, min/max, unique, required rules as "built-in" rules

```suggestion
        label="Run built-in validation rules", description="Include created built-in data validation rules in report?"
```

Reply:
fixed